evaluator/evaluate.go

```diff
@@ -117,6 +117,7 @@ func evaluateWithVmaas(ctx context.Context, tx *gorm.DB, updatesReq *vmaas.Updat
 	}
 	system.ThirdParty = thirdParty                    // to set "system_platform.third_party" column
 	updatesReq.ThirdParty = utils.PtrBool(thirdParty) // enable "third_party" updates in VMaaS if needed
+	updatesReq.OptimisticUpdates = utils.PtrBool(true)
```

Comment:
We need that for EPEL only, right? If EPEL is third-party we need to enable it only on third-party enabled.

Reply:
Fixed